.github/dependabot.yaml

```diff
@@ -12,18 +12,10 @@ updates:
     ignore:
       # These actions deliver the latest versions by updating the major
       # release tag, so ignore minor and patch versions
-      - dependency-name: "actions/*"
-        update-types:
-          - version-update:semver-minor
-          - version-update:semver-patch
       - dependency-name: "Apple-Actions/import-codesign-certs"
         update-types:
           - version-update:semver-minor
           - version-update:semver-patch
```

Comment:
If they do not release minor and patch releases. There is no point in ignoring them, right?

Reply:
True